taskcluster/script.py

```diff
@@ -4,6 +4,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+# usage: ./script.py -- ./test_command --option1 blue --option2 green
+
```

Comment:
What do you think of using argparse and https://docs.python.org/2.7/library/argparse.html#argparse.ArgumentParser.parse_known_args rather than -- ? Is there any benefit?

script.py could define --command to pass test_command and the args remaining after parse_known_args could be passed to it. This how I would have approached it but apart from "feelings" I can't really claim it would be "better". I'll leave it up to you.

Reply:
Aaah, I didn't know about that. Yeah, sounds good. Gets me out of the testing business. :)